### ISSUE TITLE
Import tweet threads

### ISSUE THREAD

**Issue by NoisyFlowers** [Nester-xyz/Porto#8]
Currently, the tool doesn't want to import full tweet threads, just the first tweet. I'm guessing this is because tweet threads are actually a series of replies to previous tweets. 

Would it be possible to change the prohibition on reply tweets to allow it to import replies to tweets by the same user?

**Comment by NoisyFlowers** [comment]
I'm not equipped to tackle this right now, but I've been looking at the source code. 

It seems like one could grab accoutJson.username in src/components/utils.jsx. Stash it somewhere (????.xUsername). Then use that to modify isPostValid in both src/lib/parse/analyze.tsx and parse.tsx from

```
  if (
    tweet.in_reply_to_screen_name ||
    tweet.full_text.startsWith("@") ||
    tweet.full_text.startsWith("RT ")
  ) {
    return false;
  }
  return true;
```

to
```
  if (
    tweet.in_reply_to_screen_name !== ????.xUsername ||
    tweet.full_text.startsWith("@") ||
    tweet.full_text.startsWith("RT ")
  ) {
    return false;
  }
  return true;
```

Maybe?

**Comment by adarshkunwar** [comment]
this would require some testing, but lets see.

**Comment by Aryog** [comment]
This issue has been solved in new version 1.2.5

**Comment by NoisyFlowers** [comment]
Excellent work on this!

My proposed solution would have brought the tweets over but not rethreaded them, as you guys have. Very nice.
